### ISSUE TITLE
CMake cannot find GMP after successful installation of CGAL using windows visual studio

### ISSUE THREAD

**Issue by lxmedai** [CGAL/cgal#7680]
## Issue Details

_I have created a cmake CGAL project using the template after successful installation of the latest CGAL in windows visual studio via vcpkg. I have included two lines to specify the included packages for CGAL as:

`set(CMAKE_CURRENT_LIST_DIR C:\\Software\\vcpkg\\packages\\cgal_x86-windows\\share\\cgal)`
`find_package(CGAL REQUIRED)`
By the way, I found out that if I didn't include the first line, CMake could not find the CGAL library. Anyhow, this is not an issue for me at present. The problem is that it produces error of not finding GMP, which is surprising for me because I can see the "FindGMP.cmake" file in the CMAKE_CURRENT_LIST_DIR as I have specified. 

The full error description generated from visual studio is given below:
```
CMake Error at C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GMP (missing: GMP_LIBRARIES GMP_INCLUDE_DIR)	
```	

## Source Code

_I have not compiled anything as GMP is not found by CMake yet._

## Environment

* Operating system (Windows 64 bits):
* Compiler: MSVC-2022
* Release or debug mode: debug
* Specific flags used (if any): not yet
* CGAL version: 5.6
* Boost version:  -
* Other libraries versions if used (Eigen, TBB, etc.): -

**Comment by lxmedai** [comment]
Update from my side, I managed to get over the problem by setting GMP_LIBRARIES and GMP_INCLUDE_DIR directly using:
`set(GMP_PREFIX C:\\Software\\vcpkg\\packages\\gmp_x64-windows)`
`set(GMP_LIBRARIES ${GMP_PREFIX}\\lib)`
`set(GMP_INCLUDE_DIR ${GMP_PREFIX}\\include)`
Then CMake complains about missing MPFR_LIBRARIES and MPFR_INCLUDE_DIR, and then I use the same way to get around it. 
Currently, it complains about missing Boost_INCLUDE_DIR, which I have not figured out which directory I should add as there are many boost packages installed in my vcpkg packages directory (C:\Software\vcpkg\packages). It seems that CMake overlooks 'CGAL_TweakFindBoost.cmake' in my CMAKE_CURRENT_LIST_DIR  directory as well. 
I would appreciate if anyone can help out with the issue that I am facing.

**Comment by lrineau** [comment]
You might want to read about the [vcpkg/CMake integration](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration) in the vcpkg documentation.

**Comment by lxmedai** [comment]
> You might want to read about the [vcpkg/CMake integration](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration) in the vcpkg documentation.

I have already tested it and it does not work. I have tried adding VCPKG_ROOT to my windows environment variable, and added the toolchain file to the CMakePresets.json as below:
    `{
            "name": "debug",
            "toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
        }`
The error I got is that find_package(CGAL REQUIRED) no longer works, and it complains about missing 'FindCGAL.cmake'.

**Comment by lxmedai** [comment]
I finally solved the problem that invokes all the above errors by install x64-windows-static version of CGAL by:
`vcpkg install cgal --triplet x64-windows-static`

Before this, I have tried setting the _toolchainFile_ in _CMakePreset.json_ as mentioned above and running `vcpkg integrate install`. I once tested trying installation of boost with **x64-windows-static** and that solved the problem of not finding boost (or missing **Boost_INCLUDE_DIR**). Then I realised that I should install cgal of the same version, which finally gave me the green light.  

In case someone might experience the same problem as I did, I share my solution here and hopefully this can be helpful or at least provide some hint.
